src/pages/AppList/FiltersAndSorts.ts

```diff
@@ -122,15 +122,17 @@ const filterByLabel = (items: AppListItem[], filter: string[]): AppListItem[] =>
   let result: AppListItem[] = [];
 
   filter.map(filter => {
-    if (filter.includes('=')) {
-      const values = filter.split('=');
+    if (filter.includes(':')) {
+      const values = filter.split(':');
       // Check Values
       values[1].split(',').map(
         val =>
           (result = result.concat(
             items.filter(item => {
               if (values[0] in item.labels) {
-                return item.labels[values[0]].split(',').some(appVal => appVal.startsWith(val));
+                return item.labels[values[0]]
+                  .split(',')
+                  .some(appVal => appVal.startsWith(val) && !result.includes(item));
```

Comment:
Perhaps it could be a good idea to trim the val, it's easy to introduce a label:     value1,  value2.
At least it would protect some cases.

Reply:
Note that trim should apply to more places where value is read, just comment on this line.